docs/common/DocsExample.vue

```diff
@@ -1,7 +1,17 @@
 <template>
 
   <div class="docs-example">
-    <div class="code-toggle-button">
+    <div
+      class="code-button"
+      :style="codeButtonStyle"
+    >
+      <KIconButton
+        v-if="loadExample"
+        appearance="raised-button"
+        icon="github"
+        tooltip="View the complete code example on GitHub"
+        @click="redirectToGitHub"
+      />
```

Comment:
Could [DocsGithubLink](https://github.com/learningequality/kolibri-design-system/blob/788c9f5f603957edb6d1074bc37e54c0d0f4ea7e/docs/common/DocsGithubLink.vue) work here?

Reply:
The same creates an anchor text link to a GitHub page, and that is only for issues and pull requests as of now. Here, we want to create a `KIconButton, ' which is not possible through that component directly. 

Though seeing that the GitHub SVG is inlined in the same, I can make either of the two changes:
- Change the `DocsGithubLink` to use this pre-compiled SVG that I have added.
- Remove the `github` icon from precompiled icons, and inline the use of the same in `DocsExample` as well. 

I personally would prefer the first option over the second, but would be happy to implement which feels to be more appropriate to the team!